app/assets/javascripts/cable/subscriptions/notifications_channel.js.coffee

```diff
@@ -0,0 +1,15 @@
+App.cable.subscriptions.create 'NotificationsChannel',
+  connected: ->
+    console.log('Subscribed to NotificationsChannel.')
+
+  rejected: ->
+    console.log('Error subscribing to NotificationsChannel.')
+
+  received: (data)->
+    $('[data-behavior~=notifications-dot]').removeClass('hidden')
+
+    $container = $('[data-behavior~=notifications-dropdown] + div')
+    $unreadCount = $('[data-behavior~=unread-count]')
+    if $container.is(':visible')
+      $('.notification-header').after(data.notification_html)
```

Comment:
we may get to this element from `data-behavior~=notifications-dropdown` too ?
No need to add an extra `data-behavior`, but the `notifications-dropdown` may be enough to cover all this references